bitbucket-server-post-build-status 0.0.10

teameh opened the pull request:
![TagCheck](https://bitrise-steplib-git-check.herokuapp.com/tag?pr=2449)

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

Fixes for https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status/issues/1 and https://github.com/teameh/bitrise-step-bitbucket-server-post-build-status/issues/2

trapacska commented:
Hi @teameh !

One step PR can contain a single file change. A single step.yml. Also it is forbidden to remove previous step versions.

Please re-share your step, without modifying the previous one.

teameh commented:
Oh, I deleted the other ones because that is what another maintainer told me last time 😄. But I guess that was because of the initial release. Anyway, I've updated the commit. Could you re-open en re-check?

teameh commented:
Oh, I deleted the other ones because that is what another maintainer told me last time 😄. But I guess that was because of the initial release. Anyway, I've updated the commit. Could you re-open en re-check @trapacska?
